api_server.py

```
@@ -81,12 +81,18 @@ def get_crew_list() -> List[Dict]:
 
 
 def get_availability(entity_id: int, table: str, now: datetime) -> Dict:
-    col = "crew_id" if table == "crew_availability" else "appliance_id"
+    # Use explicit queries with static table/column names to prevent SQL injection
+    # and satisfy strict static analysis tools (e.g. Bandit B608).
+    if table == "crew_availability":
+        query = "SELECT end_time FROM crew_availability WHERE crew_id = ? AND start_time <= ? AND end_time > ? LIMIT 1"
+    elif table == "appliance_availability":
+        query = "SELECT end_time FROM appliance_availability WHERE appliance_id = ? AND start_time <= ? AND end_time > ? LIMIT 1"
+    else:
+        logger.error(f"Unauthorized table access attempted: {table}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)